test/langtools/jdk/javadoc/doclet/testSnippetTag/TestSnippetTag.java

```diff
@@ -224,9 +224,9 @@ public String langAttribute() {
                         <span class="element-name">case%s</span>()</div>
                         <div class="block">A method.
                          \s
-                        <div class="snippet-container"><a href="#" class="snippet-copy" onclick="cop\
-                        ySnippet(this)" aria-label="Copy" data-copied="Copied!"><img src="../copy.sv\
-                        g" alt="Copy"></a>
+                        <div class="snippet-container"><button class="snippet-copy" onclick="copySni\
```

Comment:
I'm sorry to see that you had to update snippet HTML in multiple places again. Earlier I said that I was going to improve it [^1] and I've already encapsulated the HTML representation of a snippet in the context of not-yet-integrated JDK-8273544. So hopefully, next time you'll need to update snippet HTML, you'll have to do it in one place only.

[^1]: https://github.com/openjdk/jdk/pull/6011#discussion_r732034367